readthedocs/templates/projects/project_version_detail.html

```diff
@@ -16,8 +16,12 @@
 {% block content %}
     <h2> Editing {{ version.slug }} </h2>
     <form method="post" action=".">
-        {% csrf_token %}
-        {{ form.as_p }}
-        <input type="submit" value="Save">
+      {% csrf_token %}
+      {{ form.as_p }}
+      <p>
+        <input style="display: inline;" type="submit" value="Save">
+        or
```

Comment:
`Save` and `or` should be translated too

Reply:
Done, thanks!